android/tests_backend/app.py

```diff
@@ -42,6 +42,9 @@ def logs_path(self):
     def assert_app_icon(self, icon):
         pytest.xfail("Android apps don't have app icons at runtime")
 
+    def assert_dialog_in_focus(self, dialog):
```

Comment:
This (and other "empty" implementations) should either document *why* they're empty implementations, or provide a token "dialog is visible" test.